app/bundles/CoreBundle/Translations/en_US/messages.ini

```diff
@@ -15,6 +15,155 @@ mautic.core.actions="Actions"
 mautic.core.advanced="Advanced"
 mautic.core.alias="Alias"
 mautic.core.autogenerated="Autogenerated"
+mautic.core.searchcommands.api="API"
+mautic.core.searchcommands.assets="Assets"
+mautic.core.searchcommands.categories="Categories"
+mautic.core.searchcommands.dynamiccontent="Dynamic Content"
+mautic.core.searchcommands.emails="Emails"
+mautic.core.searchcommands.forms="Forms"
+mautic.core.searchcommands.leadfields="Lead Fields"
+mautic.core.searchcommands.contacts="Contacts"
+mautic.core.searchcommands.segments="Segments"
+mautic.core.searchcommands.notifications="Notifications"
+mautic.core.searchcommands.pages="Pages"
+mautic.core.searchcommands.plugins="Plugins"
+mautic.core.searchcommands.reports="Reports"
+mautic.core.searchcommands.sms="SMS"
+mautic.core.searchcommands.userroles="User Roles"
+mautic.core.searchcommands.users="Users"
+mautic.core.searchcommands.operators="Search Operators"
+mautic.core.searchcommands.heading.operators="Search Operators"
+mautic.core.searchcommands.heading.commands="Specific Search Commands"
+mautic.core.searchcommands.operators.exact_term="Find the exact term. For example, <code>+admin</code> will find 'admin' but not 'administrator'."
+mautic.core.searchcommands.operators.exclude_terms="Exclude terms. For example, <code>!lead</code> will exclude results containing 'lead'."
+mautic.core.searchcommands.operators.exact_phrase="Search for an exact phrase. For example, <code>\"John Doe\"</code> will find contacts with that exact name."
+mautic.core.searchcommands.operators.grouping="Group parts of your search. For example, <code>(email OR social)</code> will find results containing either 'email' or 'social'."
+mautic.core.searchcommands.operators.multiple_terms="Include results for either of multiple terms. For example, <code>email OR social</code>."
+mautic.core.searchcommands.operators.wildcard="Use as a wildcard. For example, <code>%technologies%</code> will find any contacts with 'technologies' in their company name."
+mautic.core.searchcommands.api.section.identification_naming="Identification and Naming"
+mautic.core.searchcommands.api.name="Allows searching for records based on their name field."
```

Comment:
The language translations should be also moved to their own bundles.

Reply:
Ok, this makes sense